### ISSUE TITLE
The PML Docker build from the SBT target does not contain the MONOSAT library

### ISSUE THREAD

**Issue by Gaudeval** [onera/pml-analyzer#20]
The `sbt docker` target mentioned in the README does not seem to pull any of the MONOSAT dependencies, the `.jar` and the `.so`. I am not quite sure if the resulting docker would support an interference analysis.

**Comment by kevin-delmas** [comment]
That is true, if we want to be perfectly clean we need to fetch the MONOSAT repository and compile it within the docker image.
This strategy is already implemented in the CI (check the github workflow).
@Gaudeval would you like me to modify the DockerFile to do that?
The compilation of the Docker image may be time consuming but at least totally self-contained

**Comment by kevin-delmas** [comment]
A new version enabling MONOSAT compilation directly in the Docker image is available.
Some other improvements need to be further discussed.
 
- Currently we are compiling PML as a librairy and we add the models within example folders
- We dismiss all sources related to dependency view (highly unstable)
- We do not include the tests within the Docker image

@Gaudeval Do you think it would be better to create a Docker image that contains all the sources ? Do you think we should include tests ? Do we consider having several sbt docker options to tune the obtained Docker ?

I prefer to discuss the matter first before doing any useless development

**Comment by kevin-delmas** [comment]
Put an empty project template (no exemple)
Embed test
